.NET/Microsoft.Recognizers.Text.NumberWithUnit/Extractors/NumberWithUnitExtractor.cs

```diff
@@ -161,7 +190,7 @@ public List<ExtractResult> Extract(string source)
                                 Type = this.config.ExtractType,
                             };
 
-                            if (prefixUnit != null)
+                            if (prefixUnit != null && !this.config.ExtractType.Contains("currency"))

```

Comment:
Please move this harcoded string to Constants and declare a method to check the extractor type so you don't need to repeat this line everywhere. 
Even better would be if there was another way to represent the type of the extractor in the config in an easier way to use.